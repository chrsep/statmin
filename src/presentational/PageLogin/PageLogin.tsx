import * as React from "react"
import FormLogin from "../../container/FormLogin/FormLogin"
import styled from "../../styled-components"

export default () => (
  <StyledLoginPage>
    <TextHero>Manage your jekyll site on GitHub with ease. </TextHero>
    <FormLayout>
      <FormLogin />
    </FormLayout>
  </StyledLoginPage>
)

const StyledLoginPage = styled.div`
  display: flex;
  flex-flow: column;
  flex-grow: 0;
  margin: 0 1.8rem;
`
const TextHero = styled.div`
  font-weight: bold;
  font-size: 2.8rem;
  margin: 5.4rem 0;
  color: ${({ theme }) => theme.white};
`
const FormLayout = styled.div`
  display: grid;
  justify-items: right;
  grid-gap: 0.8rem;
`
