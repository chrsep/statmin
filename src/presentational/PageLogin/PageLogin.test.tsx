import { shallow } from "enzyme"
import * as React from "react"
import PageLogin from "./PageLogin"

describe("Button Component", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<PageLogin />)
    expect(wrapper).toMatchSnapshot()
  })
})
