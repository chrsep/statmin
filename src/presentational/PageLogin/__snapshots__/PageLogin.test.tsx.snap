// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component renders correctly 1`] = `
<styled.div>
  <styled.div>
    Manage your jekyll site on GitHub with ease. 
  </styled.div>
  <styled.div>
    <Connect(FormLogin) />
  </styled.div>
</styled.div>
`;
